tests/js/spec/views/userFeedback/__snapshots__/organizationUserFeedback.spec.jsx.snap

```diff
@@ -1614,773 +1542,267 @@ exports[`OrganizationUserFeedback renders 1`] = `
             <div
               className="css-rejmra-PageContent e6lvex70"
             >
-              <UserFeedbackContainer
```

Comment:
The new snapshot does not have this container and is showing the no project message

Reply:
Added a project to the test org so this is now back.